tardis/io/logger/logger.py

```diff
@@ -1,9 +1,10 @@
 import logging
 import sys
-import logging
 import warnings
 import pyne.data
+
 from tardis.io.logger.colored_logger import ColoredFormatter, formatter_message
+from tardis.util.base import is_notebook
```

Comment:
Why is this added?

Reply:
Wrong import. I will remove it. It is not being used in the current implementation for the `logging_state` function.